project-base/app/web/imageResizer.php

```diff
@@ -98,6 +98,8 @@ function getImageFromUrl(string $url): void
 
     $etag = '"' . md5($image) . '"';
 
+    header('X-Robots-Tag: noindex, nofollow');
```

Comment:
`width` attribute is used in XML feed. Is it ok regarding this change?

Reply:
https://ssfwcc.prod.shopsys.cloud/content/sitemaps/domain_1_sitemap_image.products.xml Where ? :D

Reply:
https://ssfwcc.prod.shopsys.cloud/content/feeds/UNEryOmGtG_heureka_1.xml


![](https://i.imgflip.com/g8jl5.jpg)

Reply:
Changed